specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/domainservices.json

```diff
@@ -411,7 +439,6 @@
         "healthLastEvaluated": {
           "readOnly": true,
           "type": "string",
-          "format": "date-time",
```

Comment:
Remove format is not allowed. This will cause breaking change in SDK. Please fix your servise response according to https://tools.ietf.org/html/rfc3339#section-5.6

Reply:
I have reverted this format change.